packages/react-components/react-breadcrumb/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -1,16 +1,37 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
+type BreadcrumbSize = 'small' | 'medium' | 'large';
+type BreadcrumbDividerType = 'chevron' | 'slash';
+/**
+ * Data shared between breadcrumb components
+ */
+export interface BreadcrumbContextValue {
+  /**
+   * Controls size of Breadcrumb items and dividers.
+   *
+   * @default 'medium'
+   */
+  size?: BreadcrumbSize;
+
+  dividerType?: BreadcrumbDividerType;
```

Comment:
```suggestion
  size?: 'small' | 'medium' | 'large';

  dividerType?: 'chevron' | 'slash';
```

nit Let's inline types